README.md

```diff
@@ -1 +1,61 @@
 # Temporal Polyfill
+
+## Polyfill for [TC39 Proposal: Temporal](https://github.com/tc39/proposal-temporal)
+
+This polyfill is maintained by the champions of the [Temporal proposal](https://github.com/tc39/proposal-temporal) and is intended be ready for production use by the time the Temporal proposal reaches Stage 4.
```

Comment:
```suggestion
This polyfill is maintained by the champions of the [Temporal proposal](https://github.com/tc39/proposal-temporal) and is intended to be ready for production use by the time the Temporal proposal reaches Stage 4.
```
In general I think we ought to avoid committing to specific timelines. We could consider saying something else here, although "by Stage 4" seems OK enough.

We had talked in one of our previous meetings (can't remember when) about how most of the champions don't have time to do extensive maintenance on it. Maybe we should at least say it's maintained by "some of" the champions, but I wonder if there's a phrase that would be more encouraging towards people reading this to join up and help maintain it.

Consider also mentioning here that this polyfill is not associated with TC39.

Reply:
Sounds good. I revised this text. Take a look at the new text and let me know what you think.